secrets/secrets.go

```diff
@@ -278,3 +309,11 @@ func (e encoding) decodeValue(value string) (string, error) {
 		return string(data), nil
 	}
 }
+
+// ErrorSecretNotFound is returned when the key for a secret is not present in
+// the secret store.
+type ErrorSecretNotFound string
```

Comment:
style nit: The go convention is that if it's an error type, it's named `BlaBlaError`. If it's an error variable, it's named `ErrBlaBla`. So this should be `SecretNotFoundError` instead.

Reply:
Just realized this is from old code, but you could still fix it :) (but it would be a breaking change)